nym-vpn-apple/Services/Sources/Services/NymLogger/LogFileManager.swift

```diff
@@ -24,46 +34,61 @@ public final class LogFileManager {
         guard let logsDirectory else { return nil }
 
         try? fileManager.createDirectory(at: logsDirectory, withIntermediateDirectories: true, attributes: nil)
-        let logFileURL = logsDirectory.appendingPathComponent(Constants.logFileName.rawValue)
+        let fileName = "\(logFileType.rawValue)\(Constants.logFileName.rawValue)"
+        let logFileURL = logsDirectory.appendingPathComponent(fileName)
+
         return logFileURL
     }
 
-    public func logs() -> String {
-        guard let logFileURL = logFileURL,
-              let logData = try? Data(contentsOf: logFileURL),
-              let appLogs = String(data: logData, encoding: .utf8)
-        else {
-            return ""
+    public func write(_ string: String) {
+        ioQueue.async {
+            try? self.fileHandle?.write(contentsOf: Data(string.utf8))
         }
-        return appLogs
     }
 
     public func deleteLogs() {
         ioQueue.async {
-            guard let logFileURL = self.logFileURL else { return }
+            guard let logFileURL = self.logFileURL(logFileType: self.logFileType) else { return }
```

Comment:
You don't need to obtain "current" log file URL since you deleted both `.app` and `.tunnel` below.